docs/ai/conceptual/prompt-engineering-in-dot-net.md

```diff
@@ -0,0 +1,189 @@
+
+### Primary content
+
+Primary content is text you add to an instruction for the model to process as if it were user input. For example, if you add the instruction **"Summarize US Presidential accomplishments."**, you could add a list of accomplishments as primary content.
+
+Make sure you clearly separate primary content from the instructions that apply to it, such as by labeling it. Consider the following C# example.
+
+```csharp
+prompt= @$"Instructions: Summarize US Presidential accomplishments.
+Accomplishments: 'George Washington
+First president of the United States.
+First president to have been a military veteran.
+First president to be elected to a second term in office.
+First president to receive votes from every presidential elector in an election.
+First president to fill the entire body of the United States federal judges; including the Supreme Court.
+First president to be declared an honorary citizen of a foreign country, and an honorary citizen of France.
+John Adams ...'" //Text truncated;
+```
+
+### Supporting content
+
+Supporting content is text that the model processes as part of an instruction, but which isn't the subject of the instruction. The instruction must refer to the supporting content. As with primary content, supporting content should be clearly distinct from the instruction it supports.
+
+Suppose you use the instruction **"Summarize US Presidential accomplishments"** to produce a list. The model might organized and order it in any number of ways. But what if you want the list to group the accomplishments by a specific set of categories? You could adjust your instruction by appending **"&nbsp;grouped by category"** to it, but a model is unlikely to correctly determine which specific categories you want.
+
+To make sure the model uses the categories you want, you could append supporting content to specify your categories and adjust your instruction accordingly. You could add a line of supporting content below the instruction and then change the instruction so it refers to the supporting content:
+
+```csharp
+prompt = @$"Instructions: Summarize US Presidential accomplishments, grouped by category.
+Categories: Domestic Policy, US Economy, Foreign Affairs, Space Exploration.
+Accomplishments: George Washington
+First president of the United States.
+First president to have been a military veteran.
+First president to be elected to a second term in office.
+First president to receive votes from every presidential elector in an election.
+First president to fill the entire body of the United States federal judges; including the Supreme Court.
+First president to be declared an honorary citizen of a foreign country, and an honorary citizen of France.
+John Adams ..."; //Text truncated
+```
+
+## Examples
```

Comment:
Again, here I'd choose a header that makes it clear you mean examples in your prompt, not examples for the reader to try.